public/app/plugins/datasource/tempo/package.json

```diff
@@ -40,10 +40,7 @@
     "uuid": "9.0.0"
   },
   "devDependencies": {
-    "@babel/core": "7.23.7",
```

Comment:
This is the main one not in `@grafana/plugin-configs`, which I assume we don't need to build this plugin? Also `@swc/core` equivalent is `swc-loader` in `@grafana/plugin-configs`

Reply:
ahh I'm a little confused then why tslib cannot also be inherited from `@grafana/plugin-configs`... maybe because it's a dependency and not a devDependency? The main thing anyway is that the plugin builds and works correctly 👍